finagle-core/src/test/scala/com/twitter/finagle/util/CachedHashCodeTest.scala

```diff
@@ -48,8 +48,12 @@ class CachedHashCodeTest extends FunSuite {
   }
 
   test("same hashCode mixed in or not") {
-    case class NotMixedIn(s: String)
-    case class MixedIn(s: String) extends CachedHashCode.ForCaseClass
+    case class NotMixedIn(s: String) {
+      override def productPrefix = "prefix"
```

Comment:
why do we have to do this?  can you add a comment?

Reply:
Added comment above:

    //case class productPrefix (class name) gets mixed in
    //into hash in 2.13+, so we have to override
    //it if we want to check whether to separate
    //case classes would have hashed the same if
    //the only difference had been the mixin.